doc/start/os-recommendations.rst

```diff
@@ -59,6 +59,36 @@ distribution that includes a supported kernel and supported system startup
 framework, for example ``sysvinit`` or ``systemd``. Ceph is sometimes ported to
 non-Linux systems but these are not supported by the core Ceph effort.
 
+
++---------------+-----------------+------------------+------------------+
+|               | Quincy (17.2.z) | Pacific (16.2.z) | Octopus (15.2.z) |
++===============+=================+==================+==================+
```

Comment:
The backport is adding a second/duplicate table -- there already is one in the quincy branch, just formatted differently, see lines 43-55 in this file.  I'd have expected the new note to be added there.

Reply:
I'm in the middle of the process of repairing this. This backport was generate by a homemade script that I wrote a couple of years ago, and in cases like this where the information in the release branches diverges, the commits have to be hand-tooled after their generation.

Give me half an hour to get this into shape.